src/doc/unstable-book/src/compiler-flags/extern options.md

```diff
@@ -0,0 +1,19 @@
+The behavior of the `--extern` flag can be modified with `noprelude`, `priv` or `nounused` options.
+
+This is unstable feature, so you have to provide `-Zunstable-options` to enable it.
+
+## Examples
+
+Use your own build of the `core` crate.
+
+`rustc main.rs -Z unstable-options --extern noprelude:core=libcore.rlib`
+
+To use multiple options, separate them with a comma:
+
+`rustc main.rs -Z unstable-options --extern noprelude,priv,nounused:mydep=mydep.rlib`
+
+## Options
+
+* `noprelude`: Do not add the crate to the external prelude. If used, it will need to be imported using `extern crate`.
+* `priv`: Mark the crate as a private dependency for the [`exported_private_dependencies`](https://doc.rust-lang.org/stable/rustc/lints/listing/warn-by-default.html#exported-private-dependencies) lint.
+* `nounused`: Suppress [`unused-crate-dependencies`](https://doc.rust-lang.org/stable/rustc/lints/listing/allowed-by-default.html#unused-crate-dependencies) warnings for the crate.
```

Comment:
sorry, I was unclear - I'd rather not hard-code the stable channel in the docs. You can instead use relative links:
```suggestion
* `priv`: Mark the crate as a private dependency for the [`exported_private_dependencies`](https://doc.rust-lang.org/stable/rustc/lints/listing/warn-by-default.html#exported-private-dependencies) lint.
* `nounused`: Suppress [`unused-crate-dependencies`](../../rustc/lints/listing/allowed-by-default.html#unused-crate-dependencies) warnings for the crate.
```
at some point it would probably be nice to add templating for `{{channel}}`, like for rustdoc tests, but that doesn't need to block this PR.

Reply:
Oh, sorry! I checked a few other pages in the unstable book that seemed to be using absolute links too.

Reply:
> I checked a few other pages in the unstable book that seemed to be using absolute links too.

yeah, it would be nice to fix those at some point ... especially for the unstable book these can change a lot between stable and nightly.